rest/src/test/java/org/teavm/flavour/rest/test/RestIT.java

```diff
@@ -26,13 +26,20 @@
 @RunWith(TeaVMTestRunner.class)
 @SkipJVM
 public class RestIT {
+
```

Comment:
It's not something crical, but why did you add these empty lines? It's just an unnecessary change

Reply:
Agreed. Was not intentional, the IDE applied my global Java formatting on save automatically